modules/strelka/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    strelka:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id} {pair_status}
+            sample_bam: null  # UPDATE
+            sample_bai: null  # UPDATE
+            candidate_indel: null  # UPDATE
+
+        scratch_subdirectories: []
+
+        options:
+            configure:
+                mrna: "--rna"
+                capture: "--exome"
+            strelka: "-m local"
+
+        conda_envs:
+            strelka: "{MODSDIR}/envs/strelka-2.9.10.yaml"
+            tabix: "{MODSDIR}/envs/tabix-0.2.6.yaml"
+            
+        threads:
+            strelka: 4
+
+        mem_mb:
+            strelka: 2000
```

Comment:
Have you done any testing with Strelka to check its peak memory usage?

Reply:
No I haven't. 2000 seems to be enough for mRNA and capture. What did you use for your genome samples?